examples/rebalancer/src/requests/createPrivateKey.ts

```diff
@@ -1,49 +1,56 @@
+import { TurnkeyClient } from "@turnkey/http";
+import { createActivityPoller } from "@turnkey/http/dist/async";
+import { ApiKeyStamper } from "@turnkey/api-key-stamper";
 import { TurnkeyActivityError } from "@turnkey/ethers";
-import { refineNonNull } from "./utils";
 
 export default async function createPrivateKey(
   privateKeyName: string,
   privateKeyTags: string[]
 ): Promise<string> {
-  // Initialize `@turnkey/http` with your credentials
-  httpInit({
-    apiPublicKey: process.env.API_PUBLIC_KEY!,
-    apiPrivateKey: process.env.API_PRIVATE_KEY!,
-    baseUrl: process.env.BASE_URL!,
-  });
+  const turnkeyClient = new TurnkeyClient(
+    { baseUrl: process.env.BASE_URL! },
+    new ApiKeyStamper({
+      apiPublicKey: process.env.API_PUBLIC_KEY!,
+      apiPrivateKey: process.env.API_PRIVATE_KEY!,
+    })
+  );
+
+  console.log(
+    "`process.env.PRIVATE_KEY_ID` not found; creating a new Ethereum private key on Turnkey...\n"
+  );
 
-  // Use `withAsyncPolling` to handle async activity polling.
-  // In this example, it polls every 250ms until the activity reaches a terminal state.
-  const mutation = withAsyncPolling({
-    request: TurnkeyApi.createPrivateKeys,
-    refreshIntervalMs: 250, // defaults to 500ms
+  const activityPoller = createActivityPoller({
+    client: turnkeyClient,
+    requestFn: turnkeyClient.createPrivateKeys,
   });
 
   try {
-    const activity = await mutation({
-      body: {
-        type: "ACTIVITY_TYPE_CREATE_PRIVATE_KEYS_V2",
-        organizationId: process.env.ORGANIZATION_ID!,
-        parameters: {
-          privateKeys: [
-            {
-              privateKeyName,
-              privateKeyTags,
-              curve: "CURVE_SECP256K1",
-              addressFormats: ["ADDRESS_FORMAT_ETHEREUM"],
-            },
-          ],
-        },
-        timestampMs: String(Date.now()), // millisecond timestamp
+    const activity = await activityPoller({
+      type: "ACTIVITY_TYPE_CREATE_PRIVATE_KEYS_V2",
+      organizationId: process.env.ORGANIZATION_ID!,
+      parameters: {
+        privateKeys: [
+          {
+            privateKeyName,
+            privateKeyTags,
+            curve: "CURVE_SECP256K1",
+            addressFormats: ["ADDRESS_FORMAT_ETHEREUM"],
+          },
+        ],
       },
+      timestampMs: String(Date.now()), // millisecond timestamp
     });
 
-    const privateKey = refineNonNull(
-      activity.result.createPrivateKeysResultV2?.privateKeys?.[0]
+    const privateKeyId = refineNonNull(
+      activity.result.createPrivateKeysResultV2?.privateKeys?.[0]?.privateKeyId
     );
-    const privateKeyId = refineNonNull(privateKey.privateKeyId);
-    const address = refineNonNull(privateKey.addresses?.[0]?.address);
+
+    const keyInfo = await turnkeyClient.getPrivateKey({
+      organizationId: process.env.ORGANIZATION_ID!,
+      privateKeyId,
+    });
+
+    const address = refineNonNull(keyInfo.privateKey.addresses[0]?.address);
```

Comment:
Same here, we can save a call to `getPrivateKey` now that V2 results contain the address!